packages/cfd/src/Containers/ctrader-derivx-trade-modal.tsx

```diff
@@ -184,17 +219,17 @@ const TradeModal = ({
                                     platform='dxtrade'
                                     onClick={() => {
                                         const account_type = getCFDAccountKey({
-                                            market_type: mt5_trade_account.market_type,
-                                            sub_account_type: mt5_trade_account.sub_account_type,
+                                            market_type: ctrader_derivx_trade_account.market_type,
+                                            sub_account_type: ctrader_derivx_trade_account.sub_account_type,
                                             platform: CFD_PLATFORMS.MT5,
```

Comment:
shouldn't it be dxtrade?
```suggestion
                                            platform: CFD_PLATFORMS.DXTRADE,
```